examples/STM32L475VG-DISCOVERY-IOT/BTLE_sensors_tof_demo/BTLE_sensors_tof_demo.ino

```diff
@@ -0,0 +1,427 @@
+/*
+
+ BTLE_sensors_tof_demo
```

Comment:
what means tof ?

Reply:
Time Of Flight.
Perhaps I need to rename the demo file. I can use BTLE_sensors_demo.ino as the Time Of Flight is "just" a sensor...